src/WarcraftLegacies.Source/Quests/BlackEmpire/QuestBladeoftheBlackEmpire.cs

```diff
@@ -0,0 +1,40 @@
+﻿using MacroTools.ArtifactSystem;
+using MacroTools.FactionSystem;
+using MacroTools.LegendSystem;
+using MacroTools.ObjectiveSystem.Objectives.ControlPointBased;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.QuestSystem;
+
+namespace WarcraftLegacies.Source.Quests.BlackEmpire
+{
+
+  public sealed class QuestBladeoftheBlackEmpire : QuestData
+  {
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="QuestBladeoftheBlackEmpire"/> class.
+    /// </summary>
+    public QuestBladeoftheBlackEmpire() : base("The Blade of the Black Empire",
+      "An ancient blade from the time of the Black Empire itself. The Dagger has a mind of its own and tries to corrupt whoever wields it. It has been lost to time.",
+      @"ReplaceableTextures\CommandButtons\BTNmidnightGS.blp")
+    {
+      AddObjective(new ObjectiveControlLevel(UNIT_N00P_THE_ABYSS, 20));
```

Comment:
Lorewise it's a bit rough - I don't think there's a great reason for the knife to be here. But I don't think the canonical location of the Tirisfal Glades works particularly well either. So I'll mark this as resolved.